gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -515,6 +526,12 @@ public int read(ByteBuffer byteBuffer) throws IOException {
     } catch (InterruptedException e) {
       cancelCurrentRequest();
       throw new IOException(e);
+    } finally {
+      long endTime = System.currentTimeMillis();
+      logger.atFinest().log(
+          "Read request for resource %s responded/errored after %d milliseconds,"
```

Comment:
I suggest have separate logs for success and exception.

Reply:
Done